metadata-ingestion/src/datahub/ingestion/source/powerbi/config.py

```diff
@@ -195,6 +200,12 @@ def workspace_id_backward_compatibility(cls, values: Dict) -> Dict:
         return values
 
 
-class PowerBiDashboardSourceConfig(PowerBiAPIConfig):
+class PowerBiDashboardSourceConfig(
+    PowerBiAPIConfig,
+    LineageStatefulIngestionConfig,
+    UsageStatefulIngestionConfig,
+    ProfilingStatefulIngestionConfig,
+    StatefulStaleMetadataRemovalConfig,
```

Comment:
remove these, and add `StatefulIngestionConfigBase`

then add this to the config 
```
    stateful_ingestion: Optional[StatefulStaleMetadataRemovalConfig] = pydantic.Field(
        default=None, description="PowerBI Stateful Ingestion Config."
    )
```